cluster/proto/api/message.proto

```diff
@@ -114,7 +114,10 @@ message TenantProcessRequest {
   }
   string node = 1;
   Action action = 2;
-  TenantsProcess process = 3;
+  repeated TenantsProcess tenants_processes = 4;
+    
+  TenantsProcess process = 3 [deprecated = true];
```

Comment:
Do we want to take advantage of the fact that we haven't released yet to allow breaking changes ? Just a suggestion, both approaches are fine by me

Reply:
yeah i was going back and forth on it. but i remember we had 1 RC release
giving that you also had similar thought . let's just delete it. i will push the change